src/l3.c

```diff
@@ -36,6 +44,14 @@
 #define L3_STATIC_ASSERT    _Static_assert
 #endif
 
+#if __APPLE__
+#define L3_THREAD_LOCAL
+#define L3_GET_TID()  0
```

Comment:
So we'll always report a tid of 0 on Mac? That seems unfortunate (see above).

Reply:
Have fixed it, like so:

```
+#define L3_THREAD_LOCAL __thread
+#define L3_GET_TID()    pthread_mach_thread_np(pthread_self())
```